### PR TITLE
Initial services integration

### PR DESCRIPTION
**Pull request by guillaumev**

### CODE REVIEW

File: src/app/components/service/service.service.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('app.service')
+    .factory('Service', Service);
+
+  Service.$inject = ['$resource', '$http', '$location', 'config'];
```
Comment (amyvbenson) [inline src/app/components/service/service.service.js]: $http and $location aren't used

Comment (guillaumev) [reply src/app/components/service/service.service.js]: Good point, I just fixed it

File: src/app/components/service/services.controller.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  function ServicesCtrl ($scope, $routeParams, $q, gettextCatalog, alertService, Service) {
+    $scope.request = $routeParams;
+    $scope.totalItems = 0;
+    $scope.itemsPerPage = 10;
+    $scope.currentPage = 1;
+    $scope.request.limit = $scope.itemsPerPage;
+    $scope.request.offset = 0;
+    $scope.request.sort = 'name';
+
+    var queryCallback = function (clients, headers) {
```
Comment (amyvbenson) [inline src/app/components/service/services.controller.js]: Could you use function names that are more descriptive? There are a lot of queryCallback functions already around the app. Maybe use something like setTotalServices here instead?

File: src/app/components/service/services.controller.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('app.service')
+    .controller('ServicesCtrl', ServicesCtrl);
+
+  ServicesCtrl.$inject = ['$scope', '$routeParams', '$q', 'gettextCatalog', 'alertService', 'Service'];
```
Comment (amyvbenson) [inline src/app/components/service/services.controller.js]: $q, gettextCatalog, alertService aren't used

Comment (guillaumev) [reply src/app/components/service/services.controller.js]: Fixed

File: src/app/components/service/services.controller.js (hunk 2 above)
Comment (guillaumev) [reply src/app/components/service/services.controller.js]: Fixed

**Pull request merged**